Tests/OptionalTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  OptionalTests.swift
+//  CombineExt
+//
+//  Created by Jasdev Singh on 11/05/2020.
+//  Copyright © 2020 Combine Community. All rights reserved.
+//
+
+import Combine
+import CombineExt
+import XCTest
+
+@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
```

Comment:
Tests will need to be wrapped in `#if !os(watchOS)` since CocoaPods has issues with it otherwise when deploying pods 🤷